nexuse2e-webapp/src/main/frontend/src/app/message-detail/message-detail.component.ts

```diff
@@ -81,7 +81,7 @@ export class MessageDetailComponent implements OnInit {
         partnerId: message.partnerId,
         conversationId: message.conversationId,
         messageId: message.messageId,
-        payloadId: payloadId ? payloadId.toString() : undefined,
+        payloadId: payloadId !== undefined ? payloadId.toString() : undefined,
```

Comment:
Can we use the `?.` operator?
```suggestion
        payloadId: payloadId?.toString(),
```

Reply:
Done.